components/file_limit_sticky_banner/index.tsx

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {useState} from 'react';
+import styled from 'styled-components';
+import {useIntl} from 'react-intl';
+import {useDispatch, useSelector} from 'react-redux';
+
+import OverlayTrigger from 'components/overlay_trigger';
+import Tooltip from 'components/tooltip';
+
+import {getCurrentUser, isCurrentUserSystemAdmin} from 'mattermost-redux/selectors/entities/users';
+import {get as selectPreference} from 'mattermost-redux/selectors/entities/preferences';
+import {isCurrentLicenseCloud, getSubscriptionProduct as selectSubscriptionProduct} from 'mattermost-redux/selectors/entities/cloud';
+import {savePreferences} from 'mattermost-redux/actions/preferences';
+import Constants, {CloudProducts, Preferences} from 'utils/constants';
+import {GlobalState} from 'types/store';
+import useGetUsage from 'components/common/hooks/useGetUsage';
+import {fallbackStarterLimits} from 'utils/limits';
+import useOpenPricingModal from 'components/common/hooks/useOpenPricingModal';
+import NotifyAdminCTA from 'components/notify_admin_cta/notify_admin_cta';
+import {FileSizes} from 'utils/file_utils';
+
+interface FileLimitSnoozePreference {
+    lastSnoozeTimestamp: number;
+}
+
+const snoozeCoolOffDays = 10;
+const snoozeCoolOffDaysMillis = snoozeCoolOffDays * 24 * 60 * 60 * 1000;
+const maxStarterPlanFileStorageGB = fallbackStarterLimits.files.totalStorage;
```

Comment:
Since `maxStarterPlanFileStorageGB` is actually the amount of bits allowed and not GB, and since it isn't any different than the already defined const from utils/limits, and since its only used once, can we remove this const and use the const from utils/limits directly in the place where this const is used down below?